nd4j/nd4j-backends/nd4j-api-parent/nd4j-api/src/main/java/org/nd4j/linalg/api/ops/impl/shape/OnesLike.java

```diff
@@ -55,6 +56,15 @@ public OnesLike(String name, SameDiff sameDiff, SDVariable input, DataType dataT
         this.outputType = dataType;
     }
 
+    public OnesLike(INDArray[] inputs){
+        super(inputs, null);
+    }
+
+    public OnesLike(INDArray[] inputs, DataType dataType){
+        super(inputs, null);
+        this.outputType = dataType;
+    }
```

Comment:
OnesLike takes a single input array only - we need to change the op definition, then use `super(new INDArray[]{in}, null)`